### PR TITLE
[KO] Translate Protocols

### PR DESCRIPTION
**Pull request by riseshia**
@Dalgona, @malkoG, @devleoper, @marocchino review please.

Related #956

[Origin Document](https://github.com/doomspork/elixir-school/blob/master/lessons/advanced/protocols.md)

### CODE REVIEW

File: ko/lessons/advanced/protocols.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+:string
+iex> to_atom(:an_atom)
+:an_atom
+iex> to_atom([1, 2])
+:"\x01\x02"
+iex> to_atom(%{foo: "bar"})
+:foo
+```
+
+구조체의 내부는 Map임에도 불구하고 Map의 프로토콜을 공유하지 않는다는 것은 기억할 만한 가치가 있습니다. 이들은 열거할 수 없으므로, 접근도 할 수 없습니다.
```
Comment (marocchino) [inline ko/lessons/advanced/protocols.md]: (maybe) 기억할 만한 가치가 있습니다 -> 주의해야할 점은

Comment (riseshia) [inline ko/lessons/advanced/protocols.md]: 👍

**Comment by riseshia** [comment]
ping @doomspork @devleoper

**Comment by doomspork** [comment]
Thank you @riseshia and @marocchino!

**Pull request merged**